### PR TITLE
Binaryid tracking

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

**Comment by coolo** [comment]
@adrianschroeter I think our future selves will be glad if you put a little more motivation behind this commit into the log message. I assume it's obvious to *you* what a binary_id is, but it's not to me.

File: src/api/test/functional/channel_maintenance_test.rb [hunk 1]
```diff
@@ -754,6 +754,7 @@ def test_large_channel_test
     # event handling
     UpdateNotificationEvents.new.perform
     get '/search/released/binary', params: { match: "repository/[@project = 'BaseDistro3' and @name = 'BaseDistro3_repo']" }
+puts @response.body
```
Comment (bgeuken) [inline src/api/test/functional/channel_maintenance_test.rb]: This looks like debugging code:wink:

**Comment by bgeuken** [comment]
Just wanted to write the same^^

**Comment by adrianschroeter** [comment]
yes, that is the reason why the dont merge me label is there and the commit has this particular name.

It is not supposed to be finished and more to come. Just a workaround because test suite broke locally again in a new interessting way.

File: src/api/test/functional/channel_maintenance_test.rb (hunk 1 above)
Comment (coolo) [reply src/api/test/functional/channel_maintenance_test.rb]: just ignore the 'DROP ME' commit?

File: .codeclimate.yml [hunk 2]
```diff
@@ -1,8 +0,0 @@
-version: "2"
```
Comment (hennevogel) [inline .codeclimate.yml]: Why do you delete the codeclimate configuration?

**Comment by adrianschroeter** [comment]
On Montag, 18. Februar 2019, 19:30:20 CET Henne Vogelsang wrote:
> hennevogel requested changes on this pull request.
> 
> > @@ -1,8 +0,0 @@
> 
> -version: "2"
> 
> Why do you delete the codeclimate configuration?

because the test suites would not even start atm (silly one line too long 
error).

-- 

Adrian Schroeter
SUSE Linux Products GmbH,  Maxfeldstr. 5, 90409 Nuernberg, Germany
email: adrian@suse.de

**Comment by dmarcoux** [comment]
Code Climate doesn't prevent the test suite from running. However, the `ci/circleci: checkout_code` check does.

**Comment by adrianschroeter** [comment]
got merged into https://github.com/openSUSE/open-build-service/pull/7562

**Pull request closed without merging**